README.md

```diff
@@ -1,18 +1,112 @@
 # Presenter
 
-## Requirements
```

Comment:
Would be nice to also show some badges here similar to https://github.com/Apodini/ApodiniCollector
You can use the following badge for the DOI:
[![DOI](https://zenodo.org/badge/377776151.svg)](https://zenodo.org/badge/latestdoi/377776151)